pfcp/message/send_test.go

```diff
@@ -5,6 +5,8 @@
 package message_test
 
 import (
+	"github.com/omec-project/pfcp/pfcpType"
+	"github.com/omec-project/smf/context"
```

Comment:
Please move these imports below (next to the other `github.com/omec-project/...` imports

Reply:
Done